variables.tf

```diff
@@ -127,7 +127,7 @@ variable "ee_creds_ssm_param" {
 variable "ee_pkg" {
   description = "(Optional) Filename of the Enterprise Edition package"
   type        = string
-  default     = "kong-enterprise-edition-2.3.2.0.focal.all.deb"
+  default     = "kong-enterprise-edition_2.3.2.0_all.deb"
```

Comment:
this value works fine for the Ubuntu binary but it's incorrect for the Amazon Linux binary, I think Amazon Linux should be: `kong-enterprise-edition-2.3.3.2.amzn2.noarch.rpm` using a map value here for each binary?